cdap-ui/app/cdap/styles/modals.scss

```diff
@@ -0,0 +1,91 @@
+/*
+ * Copyright © 2018 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+$button-border-color: #999999;
+$header-color: #464a57;
+$modal-spacing: 10px;
+
+.cdap-modal.modal-dialog {
+  margin-top: 125px;
+  .modal-content {
+    border-radius: 0;
+    border: 0;
+  }
+
+  .modal-header {
+    padding: 0 $modal-spacing;
+    background-color: $header-color;
+    border-bottom: 0;
+    height: 40px;
+    display: flex;
+    align-items: center;
+
+    .modal-title {
+      font-weight: normal;
+      font-size: 14px;
+      color: white;
+      line-height: 40px;
+      width: 100%;
+      .fa {
```

Comment:
I think we should be more specific for this close icon selector, just in case in the future we are going to add other icons on the title

Reply:
@ajainarayanan If you want to make it generic, let's add the class `.icon-svg` here too, which will target all the icons that use `IconSVG`

Reply:
I don't think we have a use case today, but I think it's possible in the future if we were to add logo icon for the dataprep connection for instance? But maybe for those, we'll just overwrite this.